RELEASE.md

```diff
@@ -0,0 +1,45 @@
+# Releases
+
+This file serves to provide guidance and act as a checklist for any maintainers
+to this project, now or in the future. This file should be updated with any
+changes to the process. Automated processes should be described well enough that
+they can be run in the absence of that automation.
+
+* See [CHANGELOG.md](CHANGELOG.md) for notes on versioning.
+* Fetch the latest origin branches:
+
+  ```sh
+  git fetch origin
+  git checkout master
+  git pull
+  ```
+
+* Verify that your branch matches the upstream branch:
+
+  ```sh
+  git branch --points-at=master -r  | grep origin/master >/dev/null || echo "master differs from origin/master"
+  ```
+
+* Update the `libraryVersion` constant. This is a library, so we can not assure
+  that a build flag will be used in every client that provides a compile time
+  value, let alone the correct one.
+
+  ```sh
+  vim packngo.go # change libraryVersion, "0.3.0" (no v)
+  git commit --signoff -m 'v0.3.0 version bump' packngo.go
+  ```
+
+* Tag `origin/master` with a semver tag that suits the level of changes
+  introduced:
+
+  `git tag -m "v0.3.0" -s v0.3.0 origin/master`
```

Comment:
`-a` should be good enough. None of our other tags/releases are signed which points to the complexity and hardships of dealing with GPG. `-a` makes a proper annotated tag though and is what we've been using for a long time.

Reply:
We've had this conversation before, https://github.com/packethost/packet-python/pull/95/files#diff-781a53e648f3df8d16a08ec083b04bf4R25 

I'll drop it down to `-a`, with a recommendation for `-s`, when available.